### PR TITLE
Auditing for OSS documentation

### PR DESCRIPTION
**Pull request by talSofer**

### CODE REVIEW

**Review by iddoavn — APPROVED** [review]
Looks good to me - but I would want someone else's feedback as well.

File: .secretsignore [hunk 1]
```diff
@@ -16,3 +16,4 @@ wJalrXUtnFEMI/K3MDENG/bPxRfiCYEXAMPLEKEY
 wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
 zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
 10a718b3f285d89c36e9864494cdd1507f3bc85b342df24736ea81f9a1134bcc
+-----BEGIN PRIVATE KEY-----
```
Comment (ozkatz) [inline .secretsignore:19]: is this intentional?

File: docs/enterprise/orchestration.md [hunk 2]
```diff
@@ -742,6 +742,9 @@ and audit_logs that are describing a user action (i.e create branch).
 The distinction between regular logs and audit_logs is in the boolean field log_audit.
 lakeFS and fluffy share the same configuration structure under logging.* section in the config.
 
+{: .warning }
+> The log entries marked with log_audit = true are currently available in the open-source version of lakeFS. However, please be aware that these log entries are intended to be deprecated in future releases.
```
Comment (ozkatz) [inline docs/enterprise/orchestration.md:746]:
```suggestion
> The log entries marked with `log_audit = true` are currently available in both the open-source version of lakeFS and lakeFS Enterprise. However, please be aware that these log entries are deprecated in the open source version and will be removed in future releases.
```

**Review by ozkatz — CHANGES_REQUESTED** [review]
I like the direction - I think it's the right approach! Trying to clarify that it's only deprecated for OSS and not enterprise. Not sure if I helped or made it worse :)

File: .secretsignore (hunk 1 above)
Comment (talSofer) [reply .secretsignore:19]: Yes, this file includes an example with this line that fails commits

**Review by ozkatz — APPROVED** [review]
Thank you!

**Pull request merged**